docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -38,10 +46,13 @@ We will download a focal set of Idaho sequences from GISAID's EpiCoV database.
       :width: 400
       :alt: GISAID EpiCoV Search
 
+.. |last-month| timedelta::
+   :weeks: -4
+
 2. Filter to sequences that pass the following criteria:
 
    1. From Idaho, USA
-   2. Collected within the last month (between ``2022-03-01`` and ``2022-04-01`` at the time of writing)
+   2. Collected within the last month (between |last-month| and |today| at the time of writing)
```

Comment:
Convert `image` to `figure`, add a caption explaining the static dates:

```rst
   .. figure:: ../images/gisaid-select-sequences-idaho-highlighted.png
      :width: 700
      :alt: GISAID EpiCoV filter and select sequences

      Example of GISAID filters using collection date between 2022-03-01 and 2022-04-01
```

Reply:
Do you want to update this when you update the screenshot?

Reply:
Will do!